recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module.inc

```diff
@@ -0,0 +1,11 @@
+require inc/xt_shared_env.inc
```

Comment:
please consider moving to meta-xt-images

Reply:
Done